go.mod

```diff
@@ -78,8 +78,8 @@ require (
 	kubevirt.io/client-go v0.0.0-00010101000000-000000000000
 	kubevirt.io/containerized-data-importer-api v1.60.3-0.20241105012228-50fbed985de9
 	kubevirt.io/qe-tools v0.1.8
-	libvirt.org/go/libvirt v1.11002.0
-	libvirt.org/go/libvirtxml v1.11000.1
+	libvirt.org/go/libvirt v1.11004.0
```

Comment:
Why update libvirt as part of this PR?

Reply:
v1.11 go bindings contains an update that includes the new load information also newly added on libvirt 

but I see that while this PR was open, the version was already updated to v1.11002 (during the last rebase)
I actually think we don't need this update anymore